dxr/static_unhashed/js/dxr.js

```diff
@@ -385,6 +385,8 @@ $(function() {
         if (!data.results.length) {
             resultsLineCount = 0;
             if (!append) {
+                var renderedNav = nunjucks.render('results_nav.html', data);
```

Comment:
Oh, I just meant that we could factor them up into a 2-line function and then call the function from those 3 places. I can see myself going in to fix a bug and fixing only 1 of the repeats.